handler/skills.go

```diff
@@ -127,3 +178,40 @@ func checkForId(url *url.URL) string {
 	}
 	return ""
 }
+
+/*
+extractSkillFilter() returns the filter value of the "skilltype" filter within
```

Comment:
Again, I think this is probably overkill for comments, especially on a non-exported function.  I know by the function name what this does, and the logic and error messages are self explanatory.